fusor-ember-cli/app/adapters/deployment.js

```diff
@@ -1,5 +1,12 @@
 import ApplicationAdapter from './application';
 
 export default ApplicationAdapter.extend({
-    namespace: 'fusor/api/v21'
+  namespace: 'fusor/api/v21',
+
+  urlForQueryRecord(query, modelName) {
+    if (query.deployment_id) {
+      return `/${this.get('namespace')}/deployments/${query.deployment_id}`;
+    }
```

Comment:
@cfchase, why include this in ApplicationAdapter? If we want specific logic, shouldn't we create a DeploymentAdapter? However, since we you added is the "standard convention", what is the background reason why this was added here?